src/jwt-token-issuer/Program.cs

```diff
@@ -108,6 +108,14 @@ public Task StartAsync(CancellationToken cancellationToken)
                     WriteKeyInfo(keyInfo);
 
                     WriteMessage("Token: ", _tokenHandler.WriteToken(token), ConsoleColor.Cyan);
+
+                    // Wait for user input
+                    Console.WriteLine("Press ESC to stop");
```

Comment:
what is the need for the user input?

Reply:
Previously, the .exe exited too quickly because there was no delay. This workaround keeps the command prompt open until the user presses escape or exits the command prompt.

Reply:
what was the issue that it was causing?

Reply:
The user couldn't see their token because the .exe closed too quickly.